src/clojurewerkz/elastisch/native/conversion.clj

```diff
@@ -96,20 +82,20 @@
            org.elasticsearch.action.admin.indices.flush.FlushRequest
            org.elasticsearch.action.admin.indices.refresh.RefreshRequest
            org.elasticsearch.action.admin.indices.cache.clear.ClearIndicesCacheRequest
-           org.elasticsearch.action.admin.indices.status.IndicesStatusRequest
            [org.elasticsearch.action.admin.indices.segments IndicesSegmentsRequest IndicesSegmentResponse IndexSegments]
            org.elasticsearch.action.admin.indices.alias.IndicesAliasesRequest
            org.elasticsearch.action.admin.indices.template.put.PutIndexTemplateRequest
            org.elasticsearch.action.admin.indices.template.delete.DeleteIndexTemplateRequest
-           org.elasticsearch.common.hppc.cursors.ObjectObjectCursor
+           com.carrotsearch.hppc.cursors.ObjectObjectCursor
```

Comment:
This is a weird one. I think `ObjectObjectCursor` is only imported to remove reflection warnings anyway.